Description: Use javascript packages rather than bundled files in cli.html
Author: Tom Beresford <tom.beresford@bskyb.com>
Forwarded: not-needed
Index: graphite-web/webapp/graphite/templates/cli.html
===================================================================
--- graphite-web.orig/webapp/graphite/templates/cli.html	2012-07-25 14:19:06.000000000 +0100
+++ graphite-web/webapp/graphite/templates/cli.html	2012-07-25 14:20:32.000000000 +0100
@@ -15,11 +15,11 @@
 <head>
  <link rel='stylesheet' type='text/css' href='../content/css/cli.css'>
  <link rel='stylesheet' type='text/css' href='../content/css/default.css'>
- <script src="../content/js/window/prototype.js" type="text/javascript"></script>
- <script src="../content/js/window/effects.js" type="text/javascript"></script>
+ <script src="../../../prototype/prototype.js" type="text/javascript"></script>
+ <script src="../../../javascript/scriptaculous/effects.js" type="text/javascript"></script>
  <script src="../content/js/window/window.js" type="text/javascript"></script>
  <script src="../content/js/window/window_effects.js" type="text/javascript"></script>
- <script src="../content/js/scriptaculous/scriptaculous.js?load=controls,builder"></script>
+ <script src="../../../javascript/scriptaculous/scriptaculous.js?load=controls,builder"></script>
  <script src="../content/js/cli.js" type="text/javascript"></script>
 </head>
 
